source/upgrade/upgrade_notes.rst

```diff
@@ -4,6 +4,15 @@
 Upgrade notes
 *************
 
+20.01
+=====
+
+* The default protocol configured for consul is now ``HTTP`` instead of ``HTTPS`` since it's only
+  available on ``localhost``. The ``HTTPS`` remains available via the port ``8501``.
```

Comment:
we need to change the installer to implement this behaviour. Right now it's either 8500/http or 8501/https. Not both.

Reply:
On upgrade the config file from ansible will be erased by the postinst of wazo-consul-config
https://github.com/wazo-platform/wazo-consul-config/blob/master/debian/wazo-consul-config.postinst#L54
Maybe we should discuss of the wanted behavior

Reply:
I implemented this default behavior in Ansible: https://github.com/wazo-platform/wazo-ansible/pull/60